object_detection/package.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>object_detection</name>
+  <version>0.0.0</version>
+  <description>The object_detection package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="erik@todo.todo">erik</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```

Comment:
Adding an email you can be reached by is a good idea for people who may work on this in the future :smile: Also we use the MIT license.